pandas/tests/groupby/methods/test_corrwith.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+from pandas import (
+    DataFrame,
+    Index,
+    Series,
+)
+import pandas._testing as tm
+
+
+def test_corrwith_with_1_axis():
+    # GH 47723
```

Comment:
nitpick id love to standardize on GH#xyz for greppability purposes

Reply:
Agreed - but I think not here. I'll open an issue. For grepping, I'm typically just looking for a certain number and don't come up with really any false positives that I can recall. Are you doing something differently?

Reply:
I opened #55461